apps/gauzy/src/app/pages/employment-types/employment-types.component.scss

```
@@ -1,10 +1,13 @@
 @use 'gauzy/_gauzy-dialogs' as *;
+@use '../tabset-actions' as *;
 @forward '../expenses/expense-categories/expense-categories.component';
 
 :host {
     height: 100%;
     nb-tab.content-active {
-      height: calc(100vh - 16.75rem);;
+      // NOTE: height is not set here — `tabset-fills-height` below lets the body
```

Stylelint (17.14.0)

[error] Unknown word NOTE (CssSyntaxError)

(CssSyntaxError)